src/Board.hpp.orig

```diff
@@ -0,0 +1,51 @@
+#pragma once
```

Comment:
Do not commit this file

Reply:
Adres it during rebase, will delete it today evening

Reply:
If it goes about new chages, i've rebased US7 from/with master, to avoid conflict while merging.
It's Kuba advice to keep branch always actuall.